tools/get_cross_repo_hashes.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""See USAGE for details.
+
+Potential script improvements:
+- AUTOMATE: fenix check out -> a-c hash
+- (later) Support more than nightly
+- (later) Would this be more usable as a website?
+  Would require automating extraction of a-c hash
+  from Fenix apk or other mechanism
```

Comment:
That's interesting. I was also thinking about a website some time ago. I was wondering whether we could use the treeherder API to find releases and maybe have our build attach some more information about the builds as an artifact to the builds tasks.

Reply:
For the purposes of finding a valid set of revisions for fenix -> ac -> mc, I was thinking it would be a website that takes the latest Nightly build and creates the mapping, e.g.
```
2020.09.08 -> <fenix-hash> <ac-hash> <mc-hash>
2020.09.07 -> <fenix-hash> <ac-hash> <mc-hash>
```

> maybe have our build attach some more information about the builds as an artifact to the builds tasks.

@pocmo Do you mean we'd attach something like the to the Nightly artifact Taskcluster page? e.g. https://firefox-ci-tc.services.mozilla.com/tasks/index/mobile.v2.fenix.nightly.2020.09.11.latest/arm64-v8a ? I think that would address the problem and avoid making a separate, hard-to-remember website. 😅